src/main/java/org/apache/sling/installer/factory/model/impl/FeatureModelInstallerPlugin.java

```diff
@@ -102,6 +104,9 @@
 
     private final File storageDirectory;
 
+    @Reference
+    private volatile List<ExtensionHandler> extensionHandlers;
+
```

Comment:
I assume "volatile" makes this dynamic?

Reply:
Yes, as per Declarative Services spec. However I forgot that it's not greedy by default. So I added explicit selectors for this now.